src/python/ensembl/production/hive/datafile/parsers.py

```diff
@@ -0,0 +1,325 @@
+#  See the NOTICE file distributed with this work for additional information
```

Comment:
This is not a "hive" module, in case it could be reused outside of `hive` context, I'll move this in another module say for instance `ensembl/production/datafile/`. IF you think it can be used outside of production pipeline, it could even be a dedicated python module/repo

Reply:
I think ensembl-production is the best place for this code at the moment. Those modules can be moved outside `hive`. Need to be careful with naming as `ensembl.production.datafile` namespace is already used by DataFile Toolchain libs. So I'd like to avoid clashes in case ensembl-production and datafile-toolchain would end up being installed in the same env.

Reply:
ok. We'll for moving things around another time.